pkg/workceptor/controlsvc.go

```diff
@@ -391,6 +392,8 @@ func (c *workceptorCommand) ControlFunc(ctx context.Context, nc controlsvc.Netce
 		if c.subcommand == "cancel" {
 			err = unit.Cancel()
 		} else {
+			unit.UpdateBasicStatus(5, "released work", 0)
+			time.Sleep(time.Second * 3)
```

Comment:
@AaronH88 How long after setting the status to release should we wait before removing the work units dirs? This will block the work release command from completing. I'm not sure if 3 seconds is long enough but I also don't want the release command to take too long.

Reply:
Lets make it configurable with a default of 3 seconds?